README.md

```diff
@@ -0,0 +1,150 @@
+<p align="center">
+  <img src="https://github.com/Kaiserabbas/motorcycle_booking_front_end/assets/123939543/fc0f4562-55c4-4e30-a784-6004fcb1f0e7" width="250px" alt="moto_logo">
+</p>
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Back-end API Project](#back-end)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 Motorcycle Booking Front-end <a name="about-project"></a>
+
+
+https://github.com/Kaiserabbas/motorcycle_booking_front_end/assets/123939543/d2f8989e-2e34-41d3-89bb-2d0658cb0e26
+
+
+**Motorcycle Booking Front-end** This repository contains the React/Redux front-end application for the "Motorcycle Booking" project. The front-end provides an intuitive user interface for interacting with the booking system. Users can log in with a username, browse a list of motorcycles, view detailed information about specific motorcycles, reserve appointments, add new motorcycles, and manage existing ones. The application is designed to be responsive, offering a seamless experience on both desktop and mobile devices. The front-end communicates with the back-end API to ensure a cohesive and efficient user experience.
+
+## 🛠 Built With <a name="built-with"></a>
+React, redux, node, Visual Studio Code, WSL
+
+### Tech Stack <a name="tech-stack"></a>
+
+- **React**
+- **Redux**
+- **WSL**
+
+### Key Features <a name="key-features"></a>
+
+- **User Interface**
+- **Responsive Design**
+- **Reservation Workflow**
+- **Motorcycle Management**
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://motobookingapp.onrender.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Setup <a name="setup"></a>
+
+In order to use this project.. Clone this repository to your desired folder by pasting this command in your command line interface:
+
+```
+  git clone https://github.com/Kaiserabbas/motorcycle_booking_front_end.git
+```
+
+  You can also clone the back-end project using the following command:
+
+```
+  git clone https://github.com/Kaiserabbas/motorcycle_booking_back_end.git
+```
+
+### Prerequisites <a name="prerequisites"></a>
+
+  Node
+  Git
+
+### Install <a name="install"></a>
+
+To install the project's dependencies, run:
+
+```
+  npm install
+```
+
+### Usage <a name="usage"></a>
+
+To run the project, execute the following command:
+
+```
+  npm start
+```
+
+### Run tests <a name="run tests"></a>
+
+To test, run:
+
+```
+  npm test
```

Comment:
- [ ] Since the last review, there are 2 tests that are not passing:

![image](https://github.com/Kaiserabbas/motorcycle_booking_front_end/assets/57421823/717c9fc6-93ea-491f-9312-3d5d687fdd8d)

Kindly fix them so all your frontend tests pass. 👍